packages/cli/package.json

```diff
@@ -124,7 +124,7 @@
     "esm": "3.1.4",
     "execa": "3.2.0",
     "expect": "29.5.0",
-    "express": "4.17.1",
+    "express": "4.17.3",
```

Comment:
I don't think we use express in CLI. Maybe this one can be removed?

Reply:
We use it in tests for a mock server